helm/chaos-mesh/templates/chaos-daemon-daemonset.yaml

```diff
@@ -57,6 +57,7 @@ spec:
       {{- if .Values.imagePullSecrets }}
       imagePullSecrets: {{ toYaml .Values.imagePullSecrets | nindent 8 }}
       {{- end }}
+      securityContext: {{ toYaml .Values.chaosDaemon.podSecurityContext | nindent 8 }}
```

Comment:
```suggestion
      {{- if .Values.chaosDaemon.podSecurityContext }}
      securityContext: {{ toYaml .Values.chaosDaemon.podSecurityContext | nindent 8 }}
      {{- end }}
```

Also, this field needs to be added in `values.yaml`:

```
chaosDaemon:
  podSecurityContext: {}
```

Reply:
To be consistent with other components, I suggest removing the pod prefix from this field. What do you think?

<img width="344" alt="image" src="https://github.com/chaos-mesh/chaos-mesh/assets/15034155/533dafff-839f-4ca3-a771-8aa8cf839bf5">

Reply:
@g1eny0ung  - thank you for the suggestion, I will update that